### ISSUE TITLE
New Nerves lesson

### ISSUE THREAD

**Issue by brain-geek** [elixirschool/elixirschool#1574]
Hey! 

Wanted to discuss possibility of the Nerves lesson.

The main difference between this and other lessons would be need for some physical hardware: card reader, SD card and Raspberry PI.

Other limitation is that something _may go very wrong_. Firmware burn includes part with overriding contents of the entire volume (it does its best to autodetect the connected SD card and use its location as a default option). Part of doc:
```
This command will attempt to automatically discover the SD card inserted in your host. This may fail to correctly detect your SD card, for example, if you have more than one SD card inserted or you have disk images mounted. 
```
While I don't believe this will cause any harm, the small possibility is there. What would be the best way to emphasize the need of checking the device where you're burning the image?

The most logical choice for the project would be rewriting simplified [blinky](https://github.com/nerves-project/nerves_examples/tree/master/blinky) from the examples - blinking LEDs is the simplest project. 

What do you think of the limitations here? While amount of people who can do this task is limited by the hardware needed, it would be somewhat expanded compared to the [Nerves Getting Started](https://hexdocs.pm/nerves/getting-started.html#building-and-deploying-firmware), therefor useful to some people.

**Comment by brain-geek** [comment]
Also, I see that @gemantzu suggested a `Add a Nerves category` [here](https://github.com/elixirschool/elixirschool/issues/1491#issuecomment-416076159). 

Any ideas what would be the lessons in this category? I can think of maybe one about installing and burning firmware, working with LEDs ([elixir-ale](https://github.com/fhunleth/elixir_ale), but what else? I may be missing some aspects.

**Comment by doomspork** [comment]
@brain-geek I think support for Nerves would awesome but unfortunately I have _very_ limited exposure to Nerves.  I would defer to others but happy to support however I can 😁 

ping @elixirschool/developers / @elixirschool/english

**Comment by gemantzu** [comment]
We can always make a single lesson and see how far we can go. I think we will reach a point where one lesson will not be enough, but this can always happen later.

**Comment by doomspork** [comment]
@brain-geek would you be open to reaching out to some of the Nerves' folks to see if they'd be interested?  It'd be a wonderful want to increase people's exposure to Nerves (the translations!)

**Comment by notactuallypagemcconnell** [comment]
@doomspork I can reach out to frank, we chat whenever we run into each other at conferences and hes always super enthusiastic and nice. No idea how much bandwidth he might have to help out or review things, but can always say hi and see whats up.

**Comment by doomspork** [comment]
That'd be awesome @notactuallypagemcconnell

**Comment by brain-geek** [comment]
Fixed by #1766 .

@notactuallypagemcconnell , if you could reach out to someone from Nerves project to have some  feedback on the current draft, it would be great.
